apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/TransactionContext.java

```diff
@@ -57,6 +58,12 @@
      */
     private final User user = new User();
 
+    @Nullable
+    private String frameworkName;
```

Comment:
Don't forget to include in `copyFrom` and `resetState`

Reply:
added